subproviders/websocket.js

```diff
@@ -7,7 +7,7 @@ const createPayload = require('../util/create-payload')
 
 class WebsocketSubprovider
  extends Subprovider {
-  constructor({ rpcUrl, debug }) {
+  constructor({ rpcUrl, debug, origin }) {
```

Comment:
perhaps keep debug as the last parameter

Reply:
It's not really a parameter, it is an object with es6 syntax. I can put it last for clarity's sake though.

Reply:
oh, doesn't matter then